src/gui/creds/httpcredentialsgui.cpp

```diff
@@ -125,11 +123,13 @@ void HttpCredentialsGui::showDialog()
     auto *contentWidget = qobject_cast<BasicLoginWidget *>(dialog->contentWidget());
     contentWidget->forceUsername(user());
 
-    // in this case, we want to use the login button
-    dialog->addLogInButton();
-
-    connect(dialog, &LoginRequiredDialog::finished, ocApp()->gui()->settingsDialog(), [this, contentWidget](const int result) {
-        if (result == QDialog::Accepted) {
+    auto *modalWidget = new AccountModalWidget(tr("Login required"), dialog, ocApp()->gui()->settingsDialog());
+    modalWidget->setAttribute(Qt::WA_DeleteOnClose);
```

Comment:
Its no longer a dialog so this should not be needed.
https://github.com/owncloud/client/blob/815f10fe57736c3fe41bc0997a588cdf372a3d3a/src/gui/accountsettings.cpp#L589-L589